docs/docs-ref-autogen/excel/excelscript/excelscript.range.yml

```
@@ -1124,7 +1124,14 @@ methods:
     uid: 'ExcelScript!ExcelScript.Range#getImage:member(1)'
     package: ExcelScript!
     fullName: getImage()
-    summary: Renders the range as a base64-encoded png image.
+    summary: >-
+      Renders the range as a base64-encoded png image.
+
+
+      **Note**: There is a known issue with `Range.getImage` that causes wrapped text or text that exceeds the cell
+      width to render on the same line without line wrapping. This makes the resulting image unreadable, since the text
+      overflows across the entire row. As a workaround, make sure the data in the range fits in each of the cells as a
+      single-line.
```

I can't find any evidence to support using a dash here, so I vote for no dash. 
```suggestion
      single line.
```